### PR TITLE
add mgr-sync to Admin Guide

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10704

Copied from Arch Guide, then edited & updated

### CODE REVIEW

File: modules/administration/pages/mgr-sync.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
+[[mgr.sync]]
+= Using ``mgr-sync``
+
+The ``mgr-sync`` tool is used at the command prompt.
+It provides functions for using {productname} that are not always available in the {webui}.
+The primary use of ``mgr-sync`` is to connext to the {scc} and retrieve product and package information for installed products.
```
Comment (mcalmer) [inline modules/administration/pages/mgr-sync.adoc:6]:
```suggestion
The primary use of ``mgr-sync`` is to connect to the {scc}, retrieve product and package information and setup channels to sync the packages to {productname} Server.
```

File: modules/administration/pages/mgr-sync.adoc [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+----
+mgr-sync [-h] [--version] [-v] [-s] [-d {1,2,3}] {list,add,refresh,delete}
+----
+
+[[mgr-sync-commands]]
+[cols="1,1,1", options="header"]
+.mgr-sync Commands
+|===
+| Command | Description | Example Use
+| list | List channels, organization credentials, or products | ``mgr-sync list``
```
Comment (mcalmer) [inline modules/administration/pages/mgr-sync.adoc:23]:
```suggestion
| list | List channels, organization credentials, or products | ``mgr-sync list channels``
```

Just `list` will show you the usage() and ask for the target.

File: modules/administration/pages/mgr-sync.adoc [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+| add | Add channels, organization credentials, or products | ``mgr-sync add <channel_name>``
```
Comment (mcalmer) [inline modules/administration/pages/mgr-sync.adoc:24]:
```suggestion
| add | Add channels, organization credentials, or products | ``mgr-sync add channel <channel_name>``
```

File: modules/administration/pages/mgr-sync.adoc [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+| refresh | Refresh the local copy of products, channels, and subscriptions | ``mgr-sync refresh``
+| delete | Delete existing SCC organization credentials from the local system | ``mgr-sync delete``
```
Comment (mcalmer) [inline modules/administration/pages/mgr-sync.adoc:26]:
```suggestion
| delete | Delete existing SCC organization credentials from the local system | ``mgr-sync delete credentials``
```

File: modules/administration/pages/mgr-sync.adoc [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+| sync | Synchronize all channels if left blank, or specify a specific channel | ``mgr-sync sync <channel_name>``
```
Comment (mcalmer) [inline modules/administration/pages/mgr-sync.adoc:27]:
```suggestion
| sync | Synchronize specified channel or ask for it when left blank| ``mgr-sync sync channel <channel_name>``
```

File: modules/administration/pages/mgr-sync.adoc [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+[cols="1,1,1,1", options="header"]
+.mgr-sync Optional Arguments
+|===
+| Option | Abbreviated option | Description | Example Use
+| help | ``h`` | Display the command usage and options | ``mgr-sync --help``
+| version | N/A | Display the currently installed version of ``mgr-sync`` | ``mgr-sync --version``
+| verbose | ``v`` | Provide verbose output | ``mgr-sync --verbose refresh``
+| store-credentials | ``s`` | Store credentials a local hidden file | ``mgr-sync --store-credentials``
+| debug | ``d`` | Log additional debugging information. Requires a level of 1, 2, 3. 3 provides the highest amnount of debugging information | ``mgr-sync -d 3 refresh``
+| no-sync | N/A | Use with the ``add`` command to add products or channels without beginning a synchronization | ``mgr-sync --no-sync add <channel_name>``
```
Comment (mcalmer) [inline modules/administration/pages/mgr-sync.adoc:42]: The subcommands have a lot of more options. Not sure how to describe them here.

Comment (Loquacity) [reply modules/administration/pages/mgr-sync.adoc:42]: That depends how important it is that we include them, I guess.

Comment (mcalmer) [reply modules/administration/pages/mgr-sync.adoc:42]: What about adding a note with some thing like. Options specific for commands can be found by calling 

`mgr-sync <command> --help`

**Pull request merged**